app/src/views/OperationalLearnings/Filters/index.tsx

```diff
@@ -67,12 +78,12 @@ function Filters() {
                         value={rawFilter.disasterType}
                         onChange={setFilterField}
                     />
-                    <SelectInput
+                    <PrimarySectorSelectInput
                         placeholder={strings.filterBySectorPlaceholder}
                         name="bySector"
                         options={options}
-                        keySelector={keySelector}
-                        labelSelector={labelSelector}
+                        keySelector={numericKeySelector}
+                        labelSelector={stringLabelSelector}
```

Comment:
we don't need to pass these props as they are overridden in the component itself. Only pass those props that are required by the component.